apps/debt/models/debt.py

```
@@ -21,11 +23,15 @@ class Debt(FullCleanOnSaveMixin, CommonInfo, models.Model):
     objects = DebtManager.from_queryset(DebtQuerySet)()
 
     class Meta:
-        verbose_name = "Debt"
-        verbose_name_plural = "Debts"
+        verbose_name = _lazy("Debt")
+        verbose_name_plural = _lazy("Debts")
 
     def __str__(self):
```

Ruff (0.14.5)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)